pallets/pools/src/tranche.rs

```diff
@@ -55,7 +61,33 @@ pub(super) type TrancheOf<T> = Tranche<
 
 /// Type that indicates the seniority of a tranche
 pub type Seniority = u32;
-pub type TrancheInput<Rate> = (TrancheType<Rate>, Option<Seniority>);
+
+#[derive(Debug, Encode, PartialEq, Eq, Decode, Clone, TypeInfo)]
+pub struct TrancheMetadata<MaxTokenNameLength, MaxTokenSymbolLength>
+where
+	MaxTokenNameLength: Get<u32>,
+	MaxTokenSymbolLength: Get<u32>,
+{
+	pub token_name: BoundedVec<u8, MaxTokenNameLength>,
+	pub token_symbol: BoundedVec<u8, MaxTokenSymbolLength>,
```

Comment:
Given the context is sufficient here, I would drop the `token_` prefixes. Leaving it up to you tho

Reply:
But the context is not given, when we parametrized the pallet in the runtimes. So I would like to keep it.

Reply:
You are forgiven, my favourite German doughnut!